README.md

```diff
@@ -1,15 +1,162 @@
-Quire
-=====
-multi-format digital publishing
--------------------------------
+![Quire logo](static/img/quire_logo_tm.png)
 
-This is the mono-repository for [Quire](https://quire.getty.edu), is a multi-format publishing framework. This multi-package repository contains:
+## About Quire 
 
-- `packages/cli`: the `quire` command-line interface
-- `starters/default`: a default starter project 
-- `themes/default`: the default publication theme
+Developed by Getty, Quire is a new, multiformat publishing tool available for immediate access and use. Quire is in a limited beta, © J. Paul Getty Trust, and not yet released as open-source software. **For a free license to use Quire, please complete [this form](http://bit.ly/quire-signup).**
 
+Quire is an ideal tool for anyone seeking to publish a beautiful, scholarly digital book optimized for visual imagery and designed to ensure its content is widely accessible and stable. 
 
-### License
+Quire books feature:
+
+- Page-level citation
+- Footnotes
+- Bibliographies
+- Figure images and image groups
+- Zooming images and maps
+- Video and audio embeds
+- Dynamic tables of contents and menus
+- Full-text search
+- Responsive web design
+- Web accessibility and SEO optimized
+- Outputs to web, PDF, and EPUB formats
+
+To learn more about Quire, please refere to our in progress [documentation](https://quire.getty.edu)
+
+Check out [the demo site](https://thegetty.github.io/quire-starter/), and [some of Getty’s books developed with Quire](http://www.getty.edu/publications/digital/digitalpubs.html)
+
+If you’re considering using Quire for the first time, please be sure to visit our [*Implementation Considerations.*](https://quire.getty.edu/documentation/implementation/) If you’re a developer please also visit the [*For Developers*](https://quire.getty.edu/documentation/for-developers/) section of our documentation.
+
+We encourage you to sign up for a [free GitHub account](https://github.com/join) to access the [Quire Community Forum](https://github.com/thegetty/quire/discussions/11). You can utilize the forum to ask and answer questions, share ideas, provide feedback, and assist your fellow Quire users. [Subscribe to our newsletter](http://eepurl.com/hg8ROf) for the latest news and events, new feature updates, and highlights from our community. 
```

Comment:
I'd link to just the main discussions homepage rather than this post. The post is called out prominently anyway, and I think it's nice to see the range of discussions posted